src/util/Filter.js

```diff
@@ -695,14 +692,44 @@ function filterApply(card, filter, inCube) {
   if (filter.category == 'artist') {
     res = card.details.artist.toLowerCase().indexOf(filter.arg.toLowerCase()) > -1;
   }
-  if (filter.category == 'is') {
+  if (filter.category == 'is' && colors) {
+    // FIXME: This is replicaated from sortutils.js so that it isn't dependent on that file
+    // and can be reused on the server
     switch (filter.arg) {
       case 'gold':
+        if (colors.length <= 1) {
+          res = false;
+          break;
+        }
+        res = true;
+        card.details.parsed_cost.forEach(function(symbol, index) {
+          if (symbol.includes('-')) {
+            res = false;
+          }
+        });
+        break;
+
       case 'hybrid':
-      case 'phyrexian':
-          let type = filter.arg.substring(0,1).toUpperCase() + filter.arg.substring(1);
-          res = cardIsLabel(card, type, 'Manacost Type');
```

Comment:
why are we not using cardIsLabel for this? Shouldn't we be consistent?

Reply:
Yes, would prefer to reuse - but I got stuck trying to figure out how to require the file. This code is just for testing until we figure out how to best share code between client and server.

Reply:
Isn't this already included? It shows you've replaced this code with something else